scripts/sysman/ras.yml

```diff
@@ -73,6 +79,20 @@ members:
       desc: "[in][out] Breakdown of error by category"
 --- #--------------------------------------------------------------------------
 type: struct
+desc: "Extension properties related to RAS error details"
+version: "1.7"
+class: $sRas
+name: $s_ras_ext_state_t
+base: $s_base_state_t
+members:
+  - type: uint32_t
+    name: numCategory
+    desc: "[in] Number of error category enumerated in function $sRasEnumRasErrorCategory()"
+  - type: "uint64_t**"
```

Comment:
do we need to add any mention about deprecating the existing static array usage? and recommend to use this mechanism going forward? that way in 2.0 we can formally remove the array and gives users time to adjust

Reply:
Added deprecating comment